operating/on_sky.rst

```diff
@@ -0,0 +1,113 @@
+   * Click ``Modulate`` (note that it will take a few minutes before the WFS begins modulating)
+   
+2. Once the telescope operator tells you that they are pointing at the target, navigate to the telescope acquisition box on the MagAO-X web interface (https://exao1.magao-x.org/#/): |image2|
+
+   * move the acquisition mirror to the ``In`` position
+   * Set ``K-mirror tracking``, ``ADC tracking``, and ``telescope T/T offload`` to ``On``.
+   * Press the ``acquire from guider`` button (it should light up red after putting in the acquisition mirror). You are now controlling the telescope.
+
+3. If observing in Halpha:
+
+   * Set ``stagebs`` on the web interface to ``Ha-ir``
+   
+Otherwise:
+      * Set ``stagebs`` to ``65-35``
+
+4. Camera Safety Precautions
+
+   *  **IMPORTANT** Set ``EM gain`` on the ``camwfsctrl``, ``camsci1ctrl``, and ``camsci2ctrl`` GUIs to ``1``. This will reduce the risk of damaging the cameras.
+
+5. You should see a star appear on ``camacq``. If the star appears out of focus, go to ``cursesINDI`` and search for ``camwfs.zNudge`` (Note:can't remember the exact command). Edit the value to apply a focus or a defocus . 
+
+   * In the lower left-hand corner of the ``Pupil Alignment`` GUI, click the button above the arrow keys until it reads ``Move Telescope`` |image3|
+   * Use the arrow keys below ``Move Telescope`` to steer the star onto the green cross in ``camacq``. PAUSE for a second or so after each move, as you are moving a large telescope. Use the central button between the arrow keys to change the step size. The star will look something like this when aligned on camacq: |image4|
+   * In the web interface, move the acquisition mirror to the ``Out`` position. Now, increase ``EM gain`` under ``camwfsctrl`` (for fainter targets an EM gain of ``600`` is sufficient). This will put more light on camwfs. 
+   * Now, look at ``camwfs``. Reduce the step size under ``Move Telescope`` and finely adjust the telescope pointing until the pupils on ``camwfs`` appear evenly illuminated.
+
+6. Closing the Loop
+
+   * In the ``OffloadingCtrl`` GUI, drag the slider to ``On`` and set the gain to ``0.1``. |Image5|
+   * In the ``holoop`` GUI, click ``Zero All`` to remove any existing loop gain settings.
+   * Raise the global gain to ``1.0``.
+   * Drag the slider from the ``Open`` to the ``Closed`` position. 
+   * Raise tip and tilt to gains of roughly ``0.1`` - ``0.2``. 
+   * In the ``OffloadingCtrl`` GUI under ``Focus``, press ``Dump``.
+   * Monitor the DM displays to ensure that the DMs are not heavily saturating (the DM will turn red when it saturates). 
+   * If the loop reopens, press ``Zero`` in the ``holoop`` GUI and ``Zero`` in ``OffloadingCtrl`` before attempting to close the loop again.
+
+On-Sky Alignment
+----------------
+
+Pupil Alignment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+1. Tweeter
+
+   * See the `F-test guide <./alignment>`__ for documentation regarding alignment of the tweeter. 
+
+2. NCPC

```

Comment:
you can't do this closed loop!  This all goes in a separate alignment section, that really should be done in lab mode on the source